server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -142,6 +150,7 @@ def __init__(self, date_part, interval, term, alias=None):
 class SQLTranslator(ABC):
     DIALECT: SQLDialect
     QUERY_CLS: Query
+    VALUE_WRAPPER_CLS: ValueWrapper
```

Comment:
```suggestion
    VALUE_WRAPPER_CLS = ValueWrapper
```

I believe it won't be defined otherwise